internal/app/machined/main.go

```diff
@@ -75,9 +81,10 @@ func main() {
 	defer recovery()
 
 	// Subscribe to events.
-	events := make(chan event.Type, 5) // provide some buffer to avoid blocking the bus
-	event.Bus().Subscribe(events)
-	defer event.Bus().Unsubscribe(events)
+	init := EventBusObserver{&event.Embeddable{}}
+	defer close(init.Channel())
```

Comment:
this might be an issue: as events are published asynchronously, there's a chance that event might be published on a channel after `Unregister()` is called, and writing to closed channel is a panic. so probably better to avoid closing the channel

Reply:
I believe closing the channel has no semantic difference for the code, so it should be safe to skip that

Reply:
In this yeah. Since it’s PID1 and if this returns then we are likely shutting down or rebooting. We can remove.